Week1/solution1.py

```diff
@@ -0,0 +1,9 @@
+number=int(input("enter any number:"))
+if number > 1:
+      for i in range(2,int (number/2)+1):
+         if(number % i)== 0:
+           print(number,"is not a prime number")
+      else:

```

Comment:
Contain proper indentation in python program.

Reply:
i think indentation is ok in this program. there is no error.

but try to use tab key instead of space.